src/app/components/creature/creature.component.html

```diff
@@ -85,13 +89,25 @@ <h5 *ngIf='(creature.skills.length > 0)'>Skills:</h5>
               <span *ngIf='(skill.bonus >= 0)'>+</span>{{ skill.bonus }}</li>
           </ul>
         </div>
-        <div class="col-xs-2" style="text-align: right;">
-          Strength: {{ creature.abilityScores.strength }}<br />
-          Dexterity: {{ creature.abilityScores.dexterity }}<br />
-          Constitution: {{ creature.abilityScores.constitution }}<br />
-          Intelligence: {{ creature.abilityScores.intelligence }}<br />
-          Wisdom: {{ creature.abilityScores.wisdom }}<br />
-          Charisma: {{ creature.abilityScores.charisma }}<br />
+        <div class="col-xs-3" style="text-align: right;">
+          Strength: {{ creature.abilityScores.strength }}
+          (<span
+            *ngIf='creature.abilityScores.calculateBonus(creature.abilityScores.strength) >= 0'>+</span>{{ creature.abilityScores.calculateBonus(creature.abilityScores.strength) }})<br />
```

Comment:
If these end up being a performance issue at some point, then you may want to make this into getter/accessor. @cvillhauer